src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java

```diff
@@ -363,25 +362,25 @@ public void i_send_delete_request_with_company_number(String companyNumber, Stri
         HttpHeaders headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
         headers.setAccept(Collections.singletonList(MediaType.APPLICATION_JSON));
-        headers.set(x_request_id, x_request_value);
+        headers.set(xRequestId, xRequestValue);
         headers.set("ERIC-Identity" , "SOME_IDENTITY");
         headers.set("ERIC-Identity-Type", "key");
         headers.set("ERIC-Authorised-Key-Privileges", "internal-app");
         var request = new HttpEntity<>(null, headers);
 
         ResponseEntity<Void> response = restTemplate.exchange(uri, HttpMethod.DELETE, request, Void.class, companyNumber, chargeId);
 
-        CucumberContext.CONTEXT.set("statusCode", response.getStatusCodeValue());
+        CucumberContext.CONTEXT.set("statusCode", response.getStatusCode().value());
     }
 
     @Then("the CHS Kafka API is invoked successfully with delete event payload {string}")
     public void chs_kafka_api_invoked(String payload) throws IOException {
 
         File file = new FileSystemResource("src/itest/resources/payload/input/" + payload + ".json").getFile();
         List<ServeEvent> serverEvents = WiremockTestConfig.getServeEvents();
-        assertThat(serverEvents.size()).isEqualTo(1);
-        assertThat(serverEvents.isEmpty()).isFalse();
-        String requestReceived = serverEvents.get(0).getRequest().getBodyAsString();
+        assertThat(serverEvents).hasSize(1);
+        assertThat(serverEvents).isNotEmpty();
```

Comment:
The second line is semantically the same as the first

Reply:
good point, I've removed it.